resources-generator/src/main/kotlin/dev/icerock/gradle/generator/factory/ImageGeneratorFactory.kt

```diff
@@ -44,7 +45,11 @@ internal class ImageGeneratorFactory(
             generator = ImageResourceGenerator(),
             platformResourceGenerator = createPlatformImageGenerator(),
             filter = {
-                include("images/**/*.png", "images/**/*.jpg", "images/**/*.svg")
+                val formats = mutableListOf("images/**/*.png", "images/**/*.jpg", "images/**/*.svg")
+                if (allowWebpImageFormat) {
+                    formats.add("images/**/*.webp")
+                }
```

Comment:
if it will be allowed - how it will works on platforms? sample should be added in `resources-gallery` and `compose-resources-gallery` samples